shared/src/business/entities/EntityConstants.js

```diff
@@ -0,0 +1,779 @@
+const courtIssuedEventCodes = require('../../tools/courtIssuedEventCodes.json');
+const documentMapExternal = require('../../tools/externalFilingEvents.json');
+const documentMapInternal = require('../../tools/internalFilingEvents.json');
+const { sortBy } = require('lodash');
+
+const SERVICE_INDICATOR_TYPES = {
+  SI_ELECTRONIC: 'Electronic',
+  SI_NONE: 'None',
+  SI_PAPER: 'Paper',
+};
+
+const DOCKET_NUMBER_MATCHER = /^([1-9]\d{2,4}-\d{2})$/;
```

Comment:
Same here please! Short comment on what the regex is supposed to filter - and/or examples.